packages/Modal/Modal.jsx

```diff
@@ -33,6 +33,7 @@ const Modal = ({
   onClose,
   onConfirm,
   focusElementAfterClose,
+  a11yContent,
```

Comment:
What does this new prop have to do with accessibility? Is there any way we can use the existing `bodyText` for the new content?

Reply:
Nope. Actually, we are unable to achieve the same, with `bodyText`. The design that we are trying to achieve is this https://telus.invisionapp.com/d/main#/console/19991511/418659263/preview. I am unable to modify the content like adding newLine and making text bold with existing props.

I intend to add my custom component which follows accessibility guidelines (TDS-packages over here).